R/build_lookup.R

```diff
@@ -62,6 +62,41 @@ parse_imgt_fasta <- function(infile) {
   imgt_list
 }
 
+#' Extract gene names from a reference CSV for a given species
+#'
+#' `parse_mixcr_csv()` generates mixcr gene name data for a given species.
+#' @param data_dir A string, the path to directory containing mixcr CSV files.
+#'
+#' @return A character vector of gene names.
+#' @export
+#' @keywords internal
+#' @examples
+#' mixcr_dir <- get_example_path("mixcr_dir/test_mixcr")
+#' parse_mixcr_csv(mixcr_dir)
+parse_mixcr_csv <- function(data_dir) {
+  ## Throw error if the directory provided for data_dir is not valid
```

Comment:
```suggestion
  # Throw error if the directory provided for data_dir is not valid
```
Just for consistency, I suggest using one '#' for comments here and throughout

Reply:
Made the change in the most recent commit

https://github.com/seshadrilab/tcrconvertr/blob/8f63d9743f10aafb4770e70d09c7897f9c47f522/R/build_lookup.R#L77-L99